src/js/WebParts/ResourceAllocation/index.tsx

```diff
@@ -65,11 +66,11 @@ export default class ResourceAllocation extends BaseWebPart<IResourceAllocationP
         if (data.groups.length === 0 || data.items.length === 0) {
             return <MessageBar>{__.getResource("ResourceAllocation_ErrorText")}</MessageBar>;
         }
-
+        const root = this.props.rootWeb ? "../.." : "..";
```

Comment:
Er det ikke her bedre å bruke _spPageContext.siteAbsoluteUrl?

Reply:
Denne skjønner jeg ikke. Hva brukes `root` variabelen til?

Reply:
Her er det jeg som ikke har tenkt meg godt nok om. `root` variablen var brukt for å endre path til listen på rot, dersom webdelen var renderet et annet sted (programrom), men @tarjeieo har selvfølgelig rett. Skal vel kunne bare bruke siteAbsoluteUrl i begge tilfeller.

[9ee7ed3](https://github.com/Puzzlepart/prosjektportalen/pull/658/commits/9ee7ed383cd7bb82bf9299c55f49ed883bd392bf) <- Dette en grei løsning?